go.mod

```diff
@@ -12,13 +12,22 @@ exclude (
 )
 
 require (
-	github.com/google/go-cmp v0.3.1
-	github.com/hashicorp/go-multierror v1.0.0
-	github.com/hashicorp/go-version v1.2.0
-	github.com/hashicorp/hcl v0.0.0-20170504190234-a4b07c25de5f
+	github.com/google/go-cmp v0.5.0
+	github.com/hashicorp/go-multierror v1.1.0 // indirect
+	github.com/hashicorp/go-version v1.2.1
+	github.com/hashicorp/hcl v1.0.1-0.20201016140508-a07e7d50bbee // indirect
+	github.com/hashicorp/nomad v0.12.5-0.20201029140339-d6255129a300
 	github.com/hashicorp/nomad/api v0.0.0-20201028165800-38e23b62a770
-	github.com/hashicorp/terraform-plugin-sdk v1.15.0
+	github.com/hashicorp/terraform-plugin-sdk v1.16.0
+	github.com/hashicorp/terraform-plugin-test v1.4.3 // indirect
 	github.com/hashicorp/vault v0.10.4
```

Comment:
wow - that's so old!

Reply:
Old and wrong 😬 
https://github.com/hashicorp/terraform-provider-nomad/issues/150